CHANGELOG.md

```diff
@@ -78,6 +78,8 @@ This release closes the [3.0.0 milestone](https://github.com/Instagram/IGListKit
 
 ### Fixes
 
+- Fix bug - issue https://github.com/Instagram/IGListKit/issues/680 with empty space in CollectionView . [Artem Goncharov](https://github.com/madmag77)
```

Comment:
Sorry last thing, can we remove this since `IGListCollectionViewLayout` is a new feature between 2.1 and 3.0, we don't need an entry for it.